wpimath/src/test/java/edu/wpi/first/math/controller/LinearSystemLoopTest.java

```diff
@@ -26,6 +26,7 @@ class LinearSystemLoopTest {
   private static final double kPositionStddev = 0.0001;
   private static final Random random = new Random();
 
+  @SuppressWarnings("removal")
```

Comment:
Why are the tests using deprecated functionality? They should be ported to using the new overloads.